pipeline_standards.md

```diff
@@ -0,0 +1,42 @@
+# Pipeline Best Practices
+
+The range of pipeline projects that the ACF Data Surge team may take on will vary in scope and complexity. This guide foregoes specific recommendations in favor of providing a set of high-level considerations central to a well-executed pipeline project.
+
+## Define clear goals/outcomes 
+Understand and clearly define the business goals by knowing the answers  to the following questions:
+- What is the problem?
+    - Thoroughly understand the purpose of the pipeline. Spend significant time discussing the broad goal of the pipeline with stakeholders so that this goal is always in focus during pipeline development.
+- What data sources/destinations are needed?
+    - Clearly define where data are coming from and where they will ultimately be stored.
+- What transformations should be applied?
+    - Clarify how the data must be manipulated, if at all, en-route to its destination.
+- What outputs are expected?
+    - Define the format of data exiting the pipeline.
+- How will success be measured?
+    - Establish clear metrics for a successful pipeline.
+
+## Choose the right tools and technologies
+Depending on the data type, volume, and velocity, choose appropriate tools and technologies. For example: 
```

Comment:
It might be helpful to distinguish tools that are possible when building GFE-based pipelines vs. cloud-based pipelines.